schema-registry/rest-service/src/main/java/com/hortonworks/registries/schemaregistry/webservice/ConfluentSchemaRegistryCompatibleResource.java

```diff
@@ -312,9 +314,10 @@ public Response registerSchemaVersion(@ApiParam(value = "subject", required = tr
 
                 SchemaIdVersion schemaVersionInfo = schemaRegistry.addSchemaVersion(schemaMetadataInfo.getSchemaMetadata(),
                                                                                     new SchemaVersion(schemaStringFromJson(schema).getSchema(), null));
-                Id id = new Id();
-                id.setId(schemaVersionInfo.getSchemaVersionId());
-                response = WSUtils.respondEntity(id, Response.Status.OK);
+
+                VersionId version = new VersionId();
+                version.setId(schemaVersionInfo.getVersion());
```

Comment:
This is meant to be the id, not the version.

Reply:
see original code where it was correctly returning id originally before refactors and this PR.

https://github.com/hortonworks/registry/blob/b54d7c298bc2516982d55d340fe53a48223a7e97/schema-registry/rest-service/src/main/java/com/hortonworks/registries/schemaregistry/webservice/ConfluentSchemaRegistryCompatibleResource.java

Reply:
Makes sense, my bad in the refactoring.